pkg/backup/backup_test.go

```diff
@@ -2346,6 +2346,62 @@ func (b *fakeResticBackupper) BackupPodVolumes(backup *velerov1.Backup, pod *cor
 	return res, nil
 }
 
+func TestGetNamespacesInBackup(t *testing.T) {
+	testCases := []struct {
+		name               string
+		backup             *velerov1.Backup
+		expectedNamespaces []string
+	}{
+		{
+			name: "backup with nil include and exclude namespaces",
+			backup: &velerov1.Backup{
```

Comment:
Could these use the builders we have for backups in pkg/builder?

Reply:
I always forget the builder package.